test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {Test, console2} from "forge-std/Test.sol";
+
+import {IVotingToken} from "src/interfaces/IVotingToken.sol";
+import {VotingTokenConcrete} from "test/VotingTokenConcrete.sol";
+import {FranchiserFactory} from "src/FranchiserFactory.sol";
+import {Franchiser} from "src/Franchiser.sol";
+
+contract FranchiserFactoryHandler is Test {
+    FranchiserFactory public factory;
+    Franchiser public franchiser;
+
+    constructor(FranchiserFactory _factory) {
+        factory = _factory;
+        franchiser = new Franchiser(IVotingToken(address(factory.votingToken())));
+    }
+
+    function validAddress(address _address) internal view returns (bool valid) {
```

Comment:
`validAddress` is a bit ambiguous. Can the name articulate more of its application? `validActorAddress` feels a little better.

otherwise, i think it's important to be least restrictive. E.g. in `handler_fund`, it makes sense to assume `_delegator` is not the factory or voting token (they aren't realistic actors here), but why do we need to assume that about `_delegatee`?

Reply:
Done: https://github.com/uniswapfoundation/franchiser/pull/1/commits/d9e4f759b401d25001911c6a55c4689c50efe038

Reply:
thanks @jferas, the second half of comment is not handled yet

Reply:
Thanks! Missed that.  Done in both fund and recall hander functions. [69d2d6d](https://github.com/uniswapfoundation/franchiser/pull/1/commits/69d2d6d98ef52417ba31e8c62f0d65b991bb4aab)
The recall handler function will change to not need this in upcoming commit where we use your excellent suggestion regarding recording the fund handler function results for use by the recall function.. Thanks!